FerramAerospaceResearch/LEGACYferram4/FARWingAerodynamicModel.cs

```diff
@@ -189,7 +185,11 @@ public class FARWingAerodynamicModel : FARBaseAerodynamics, IRescalable<FARWingA
         public float GetModuleMass(float defaultMass, ModifierStagingSituation sit)
         {
             if (MassOverride is not null)
+            {
+                Fields[nameof(curWingMass)].guiActiveEditor = false;
+                Fields[nameof(massMultiplier)].guiActiveEditor = false;
```

Comment:
Keep the fields modifications in the property setter, this function should only calculate the module mass

Reply:
Having the fields modification in the old location didn't properly hide the field in the GUI. Is there another location that would work to put it?

Reply:
Probably needs to be updated `Initialization` if `OnLoad` is not called on deserialization

Reply:
`Initialization` worked, moved the Field modifications there